### PR TITLE
GRAPH 2: Server Node Rendering

### PR DESCRIPTION
**Pull request by arniebradfo**
This is part 2 of a series of stacked PRs meant to isolate issues if they arise. #116 

## Description ##
In the Graph vis, this pr makes the server nodes function more similarly to beacon nodes
- edit the selection logic in `HierarchicalGraphData.addToSet()` to make and exception for `node.data.isServer`. If the node is part of a server, select all the ancestors and descendants in unison, as if they were a single node.
  - this fixes a bug in RedEye where the 'faux' beacon of a server could be selected and created an error state in the UI.
  - This makes a more intense visual distinction between hosts and servers for better graph comprehension
- the server renders as a hexagon for more visual distinction between hosts
- the graph displays fewer labels to be less overwhelming, preferring to show host names and not beacon names.
- the z-indexing of svg layers is now dynamic so that selected and previewed content overlays other content.

## Testing ##
- Does selecting the servers still work? 
  - from the graph?
  - from navigation? breadcrumbs? search?

## Screenshots ##
Screen Shot 2023-03-31 at 9 40 23 AM
Screen Shot 2023-03-31 at 9 40 30 AM
Screen Shot 2023-03-31 at 9 40 42 AM
Screen Shot 2023-03-31 at 9 40 57 AM

### CODE REVIEW

**Review by GoldingAustin — APPROVED** [review]

**Comment by GoldingAustin** [comment]
@arniebradfo This is good to merge after the conflicts have been resolved

**Comment by arniebradfo** [comment]
Need a review from @sang2925 or @ccarpenter28 before merging.

**Review by ccarpenter28 — APPROVED** [review]

**Pull request merged**
